lib/config/definitions.ts

```diff
@@ -140,6 +140,15 @@ const options: RenovateOptions[] = [
     admin: true,
     cli: false,
   },
+  {
+    name: 'onboardingCommitMessage',
+    description:
+      'Change this value in order to override the default onboarding commit message.',
+    type: 'string',
+    default: null,
```

Comment:
Could be more readable if we put our default here? i.e. `Add renovate.json`. However the filename is configurable (I think, never tried it) so it may be hard.

Reply:
Yes, I think you're right.